src/main/java/stringCalculator/domain/Positive.java

```diff
@@ -0,0 +1,16 @@
+package stringCalculator.domain;
+
+public class Positive {
+
+    private int number;
+
+    public Positive(String numberString) {
+        this.number = Integer.parseInt(numberString);
+        if (this.number < 0) throw new RuntimeException();
```

Comment:
- if문 안의 코드가 한 줄이더라도 중괄호를 명시해주시는 걸 추천드립니다.
중괄호를 추가함으로써 얻을 수 있는 이점은 어떤 게 있을까요?
- 예외가 발생한 원인에 대한 메시지도 같이 전달해보면 어떨까요?
```suggestion
        if (this.number < 0) {
            throw new RuntimeException("음수는 허용되지 않습니다.");
        }
```